rtsp-to-web/Dockerfile

```diff
@@ -1,47 +1,54 @@
 # syntax=docker/dockerfile:1
 #
 # Example build command from add-on root directory:
-# $ docker build --build-arg BUILD_FROM="homeassistant/amd64-base:latest" --build-arg "BUILD_ARCH=amd64" --build-arg TEMPIO_VERSION=2021.09.0 -t rtsp-to-web rtsp-to-web
-
-ARG BUILD_FROM
+# $ docker build --tag rtsp-to-web rtsp-to-web
 
 # https://developers.home-assistant.io/docs/add-ons/configuration#add-on-dockerfile
-FROM $BUILD_FROM
-ARG TEMPIO_VERSION=2021.09.0
-ARG BUILD_ARCH
+ARG BUILD_ARCH="amd64"
+ARG BUILD_FROM="ghcr.io/hassio-addons/base/${BUILD_ARCH}:11.0.1"
 
-# Execute during the build of the image
-RUN \
-    curl -sSLf -o /usr/bin/tempio \
-    "https://github.com/home-assistant/tempio/releases/download/${TEMPIO_VERSION}/tempio_${BUILD_ARCH}"
+ARG GO111MODULE="on"
+ARG GIN_MODE="release"
 
 
-RUN apk add go git
+FROM ${BUILD_FROM} AS build
 
-ARG REPO=github.com/deepch/RTSPtoWeb
-WORKDIR /build/$REPO
+RUN apk add --no-cache go=~1.17.4-r0 git=~2.34.1-r0
```

Comment:
I only added this because of Hadolint, and because every Hass Community addon also does.

See: https://github.com/hadolint/hadolint/wiki/DL3018

I can remove this, but it's known that Go 1.18 can potentially break applications meant to be built with Go 1.17.

For Git, I agree. Actually, I can easily refactor to drop the dependency on Git if you would like.

Reply:
I unpinned git version. And forget what I said about dropping the dependency on it. I forgot we also call `git apply`.